contrib/opencensus-ext-azure/opencensus/ext/azure/statsbeat/statsbeat.py

```diff
@@ -43,6 +43,7 @@ def collect_statsbeat_metrics(options):
             exporter = MetricsExporter(
                 is_stats=True,
                 connection_string=_get_stats_connection_string(options.endpoint),  # noqa: E501
+                enable_local_storage=options.enable_local_storage,
```

Comment:
This looks more like enabling storage for the exporter if it is enabled for statsbeat. Bit confused by the title. Was local storage previously not ever a part of the exporter?